docs/swagger/spec.yml

```diff
@@ -321,6 +476,89 @@ definitions:
         type: string
         description: URL safe version of name
         readOnly: true
+  LayerSource:
+    allOf:
+      - $ref: '#/definitions/BaseModel'
```

Comment:
This is actually a front-end-only model that is necessary because the schema used to create a Layer is different than the schema returned by the list and detail views, which will include information about associated tasks, so I don't think `createdBy` or `modifiedBy` should be here since we don't want to allow clients to edit those values. The name is definitely confusing though so I'll rework that.